OWNERS

```diff
@@ -1,43 +1,32 @@
 approvers:
   - Luap99
   - TomSweeneyRedHat
-  - alexlarsson
   - ashley-cui
```

Comment:
Any reason to keep @ashley-cui?

Reply:
I'm still planning on being back in Dec, if that changes anything

Reply:
I am fine to keep you here, I guess my main understanding is that I would like to only have "active" people listed. Mainly because AFAIK the bot recommends contributors to assign the PR to one approver randomly, i.e. https://github.com/containers/podman/pull/24170#issuecomment-2393967924
While most contributors do not use that for the ones that do it seems pointless to suggest people that are not reviewing the PRs anyway.